deploy/terraform-datacommons-website/modules/gke/README.md

```diff
@@ -0,0 +1,23 @@
+# GKE module
+
+This module provides the GCP resources GKE clusters and its initial setup.
+
+## Usage
+
+```tf
+module "gke" {
+  source                   =  "../../modules/gke"
+  project_id               = "my-project-id"
+}
+```
+
+Note: Change the source path if the caller module is not from a particular example in the examples folder.
+
+## Inputs
+
+| Name         | Description                              | Type     | Default       | Required |
+| ------------ | ---------------------------------------- | -------- | ------------- | :------: |
+| project_id   | GCP project id to deploy the cluster to. | `string` | n/a           |   yes    |
+| cluster_name | Name of the GKE cluster to create.       | `string` | `datacommons` |    no    |
```

Comment:
Since we have multi-cluster setup. Should the cluster_name have region name, by default "us-central1"?

Reply:
Good point. Changed cluster_name to cluster_prefix and will use the region as suffix